src-ui/app/components/device-list/device-list-item/device-list-item.component.ts

```diff
@@ -62,6 +62,13 @@ export class DeviceListItemComponent implements OnInit {
     return 'unknown';
   }
 
+  protected get deviceRole(): string | null {
+    if (this.ovrDevice && this.ovrDevice.handleType) {
+      return 'comp.device-list.deviceRole.' + this.ovrDevice.handleType;
```

Comment:
I would probably delegate this translation prefix to the template (I'll post another comment there to show how). Since this method is just named "deviceRole" it should imo be expected to return just that and nothing more.